### PR TITLE
DEV-42: Add icon component

### PR DESCRIPTION
**Pull request by welitonderesende**

### CODE REVIEW

File: src/components/Icon/Icon.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  color?: string
+
+  // set the thickness stroke
+  stroke?: number
+}
+
+const Icon = ({
+  name = "IconAlertTriangle",
+  size = 20,
+  color = "blue",
```
Comment (edagonis) [inline src/components/Icon/Icon.tsx:24]: Maybe should be using a color from the theme as the default color

**Review by edagonis — DISMISSED** [review]

File: src/components/Icon/Icon.tsx (hunk 1 above)
Comment (welitonderesende) [reply src/components/Icon/Icon.tsx:24]: that makes sense, gonna change it

**Review by edagonis — DISMISSED** [review]

File: src/components/Icon/Icon.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/** @jsxRuntime classic /
+/** @jsx jsx */
```
Comment (giubatt) [inline src/components/Icon/Icon.tsx:2]: To fix errors:
```suggestion
/** @jsx jsx */
import { jsx } from "theme-ui"
```

File: src/components/Icon/Icon.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+
+#### Large Icon:
+
+```js
+
+<Icon name="IconAlertTriangle" color="highlight" size={24} stroke={2} />
+
+```
```
Comment (giubatt) [inline src/components/Icon/Icon.md:29]: Would be good to document the component a bit better, explain where the user can see all the available icon names, how they look and so on. We need to make this documentation more like a documentation and not just code examples

Comment (welitonderesende) [reply src/components/Icon/Icon.md:29]: Yeah, that makes sense, I'm going to add a better doc on it

**Review by giubatt — APPROVED** [review]

**Review by edagonis — APPROVED** [review]

**Pull request merged**